src/usr/local/buildpack/tools/v2/helm.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+function check_tool_requirements () {
```

Comment:
we can do that certainly.
In a new PR I can provide a default with the current code and for some tools like `npm` it can override the function to reside inside the `node` folder

Reply:
Sorry, I thought we are talking about the `check_tool_installed` function. But it is valid in both cases. I provide a PR for sensitive defaults.

Reply:
Done that in #302